src/audio/aria/aria_hifi3.c

```diff
@@ -158,9 +164,13 @@ void aria_algo_get_data_even_channel(struct processing_module *mod,
 			/*process 2 samples per time if ch_n is even*/
 			for (ch = 0; ch < ch_n; ch += 2) {
 				AE_LA32X2_IP(in_sample, inu, in);
-				out_sample = AE_MULFP32X2RS(in_sample, gain);
-
-				out_sample = AE_SLAA32S(out_sample, att);
+				out1 = AE_MUL32_HH(in_sample, gain);
+				out1 = AE_SRAA64(out1, shift_bits);
+				out1 = AE_SLAA64(out1, 16);
+				out2 = AE_MUL32_LL(in_sample, gain);
+				out2 = AE_SRAA64(out2, shift_bits);
+				out2 = AE_SLAA64(out2, 16);
```

Comment:
same

Reply:
The concept of the fix has changed and this problem no longer exists.